native/src/main/java/io/ballerina/stdlib/ftp/transport/Constants.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2017 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Shall we update in other places as well?

Reply:
As we have copied most of the same content from the old WSO2 repository, `wso2/transport-file`, WSO2 should have the copyrights from the beginning (i.e. 2017) to claim against potential copyrights violations that may occur after the initial copyrights claims by WSO2. In practice that may not have much impact as the licences are Apache 2.0 but that is the common practice we should follow.